.github/workflows/ci_consumption_workflow.yml

```diff
@@ -29,9 +29,6 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple -U -e .[dev]
```

Comment:
Any specific need for `--index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple`?

Reply:
Actually just copied this over from the other github workflows. We shouldn't need test.pypi for this. I can remove it